icon.schema.json

```
@@ -84,6 +84,7 @@
           "notifications",
           "people",
           "photography",
+          "poi",
```

We don't use abbreviations for any other category
```suggestion
          "point-of-interest",
```